Cargo.toml

```diff
@@ -147,6 +148,3 @@ harness = false
 [[bench]]
 name = "public_params"
 harness = false
-
-[patch.crates-io]
-sppark = { git = "https://github.com/supranational/sppark", rev="5fea26f43cc5d12a77776c70815e7c722fd1f8a7" }
```

Comment:
This makes our tests and benches bearable on M1/M2, see https://github.com/supranational/pasta-msm/issues/2